packages/js-utils/src/pick.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Implementation of the lodash "pick" function
+ *
+ * @param object Object to pick properties from
+ * @param props The properties to pick
+ * @returns An object with only the "picked" properties
+ */
+export default function pick< T, U extends keyof T >( object: T, props: Array< U > ): Pick< T, U > {
```

Comment:
Seems like we can delete this one from the PR now. Thanks for simplifying the logic 👍

Reply:
I think we can keep this in case somebody wants to use it in the future? Or we can also re-implement this function at that time 🤔

Reply:
The logic is very easy and implementable inline. That's preferable IMHO, rather than adding additional dependencies for one-liner functions. Not to mention that there's no reason to be introducing unused code. I suggest removing it.

Reply:
Okay, I've removed it

Reply:
@tyxla Okay, I've removed it!